build-support/bin/ci.sh

```diff
@@ -27,18 +27,9 @@ Usage: $0 (-h|-2fxbkmrjlpuneycitzsw)
  -j           run core jvm tests
  -l           run internal backends python tests
  -p           run core python tests
- -u SHARD_NUMBER/TOTAL_SHARDS
```

Comment:
This should have been removed from https://github.com/pantsbuild/pants/pull/7724.